components/engine/Cargo.toml

```diff
@@ -8,11 +8,14 @@ publish = false
 jemalloc = ["engine_rocksdb/jemalloc"]
 portable = ["engine_rocksdb/portable"]
 sse = ["engine_rocksdb/sse"]
+prost-codec = ["raft/prost-codec", "kvproto/prost-codec", "tikv_util/prost-codec", "prost"]
+protobuf-codec = ["raft/protobuf-codec", "kvproto/protobuf-codec", "tikv_util/protobuf-codec"]
 
 [dependencies]
-kvproto = { git = "https://github.com/pingcap/kvproto.git" }
-protobuf = "2"
-raft = "0.6.0-alpha"
+protobuf = "2.8"
+kvproto = { git = "https://github.com/pingcap/kvproto.git", default-features = false }
+raft = { version = "0.6.0-alpha" , default-features = false }
+prost = { git = "https://github.com/danburkert/prost", rev = "1944c27c3029d01ff216e7b126d846b6cf8c7d77", optional = true }
```

Comment:
so here we still need to introduce prost even we don't want to use it?

Reply:
no, it's an optional dep. We only include it if the `prost-codec` feature is supplied

Reply:
Should use release version.